package.json

```diff
@@ -47,6 +47,7 @@
     "dependencies": {
         "@erc725/erc725.js": "^0.20.0",
         "@lukso/lsp-smart-contracts": "^0.11.1",
```

Comment:
Would be good to update the version of the lsp-smart-contracts package to the latest